src/main/java/com/talentpath/shamazin/showItemPage/daos/ItemRepository.java

```diff
@@ -14,4 +14,5 @@
 public interface ItemRepository extends JpaRepository<Item, Integer> {
     Optional<Item> findById(Integer itemID);
 
+    List<Item> findAllByItemFamilyId(Integer ItemFamilyId);
```

Comment:
Hey Nathaniel, was thinking 
 -- Optional<List<Item>> findAllByItemFamilyId(Integer ItemFamilyID); 
May be the case that certain families don't have any items. I know in the real world, that shouldn't really happen. Each family should have at least one item but may be the case someone forgets to write some items for itemFamily

Reply:
I think it might be ok to do it Nathanial's way because in that case it would just return an empty list. Not sure though so its definitely something to consider

Reply:
I just tried to set it up to return an Optional<List<Item>>. If there is no item for that family, calling .isPresent() on the result will still return true, and .get() will give you an empty list. Even if there is no family with that id, it will still return an empty list.